src/backend/common/helpers/insights_helper.py

```diff
@@ -226,6 +235,58 @@ def _createInsight(self, data: Any, name: str, year: Year) -> Insight:
             data_json=json.dumps(data),
         )
 
+    @classmethod
+    def _create_leaderboard_from_dict_counts(
+        cls, dict: Dict[TeamKey, int], insight_type: int, year: int
+    ) -> Insight:
+        sorted_leaderboard_tuples = cls._sortTeamWinsDict(dict)
+        leaderboard_data = [
```

Comment:
Can we type this (e.g. `leaderboard_data: LeaderboardData = []`) just to make sure we have a Python type for this and don't accidentally break this in the future?